distribution/docs/src/main/resources/content/_installing/java-reqs.adoc

```diff
@@ -70,24 +75,20 @@ setx JAVA_HOME "<PATH_TO_JAVA><JAVA_VERSION>"
 setx PATH "%PATH%;%JAVA_HOME%\bin"
 ----
 +
-. Restart Terminal (shell) or Command Prompt.
-
-* Verify that the `JAVA_HOME` was set correctly.
-====
-
-====
-
+. Restart or open up a new Terminal (shell) or Command Prompt to verify `JAVA_HOME` was set correctly. System restart is not needed for the changes to take effect.
```

Comment:
How about "It is not necessary to restart the system to for the changes to take effect".

Reply:
changed